docs/resources.rst

```diff
@@ -145,5 +145,5 @@ Third-Party Solidity Parsers and Grammars
 * `solidity-parser <https://github.com/ConsenSys/solidity-parser>`_
     Solidity parser for JavaScript
 
-* `Solidity Grammar for ANTLR 4 <https://github.com/federicobond/solidity-antlr4>`_
+* `Solidity Grammar for ANTLR 4 <https://github.com/solidityj/solidity-antlr4>`_
```

Comment:
Actually we should remove this, since we have subsumed it into the docs, don't we?

Reply:
Done.